src/blocks/big-search.css

```diff
@@ -0,0 +1,45 @@
+.big-search__input {
+  font-size: 16px;
+  margin: 0 auto;
+  width: 100%;
+  text-indent: 62px;
+  border: none;
+  background-image: url(../images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: 19;
+  background-position: 26 24;
```

Comment:
The 'background-position' property is missing units. Both values should be specified in a valid CSS unit like 'px', 'em', etc.